src/closures.rs

```diff
@@ -116,14 +120,18 @@ fn get_inner_expr<'a>(
     expr
 }
 
+fn iter_stmts_without_empty(stmts: &[ast::Stmt]) -> impl Iterator<Item = &ast::Stmt> {
+    stmts.iter().filter(|x| !matches!(x.kind, StmtKind::Empty))
+}
```

Comment:
nit: I'd really prefer if you used `stmt` instead of `x` in the closure, though if you want to use a single letter variable name then `s` feels more appropriate.